### PR TITLE
i18n issues

### PR DESCRIPTION
**Pull request by DragonZX**
Some i18n typos fix
P.S. Is Software depot = Software repsitory = Software webdepot?

### CODE REVIEW

**Comment by nuxwin** [comment]
@DragonZX 

Thank  for the review. We will merge your changes after the 1.4.8 release. It is too late to push all those changes because this would mess transifex.

Regarding your question, I would say: Web software repository (a repository that provides softwares for the Web) but that is only my personal thinking. Maybe the initial @TheCry could give you the right answer.

File: gui/library/Functions/Input.php [hunk 1]
```diff
@@ -397,7 +397,7 @@ function getUserLoginDataForm($usernameRequired = true, $passwordRequired = true
                 ]
             ],
             'admin_pass_confirmation' => [
-                'password', ['validators' => [['Identical', true, ['admin_pass', 'messages' => tr('Passwords do not match.')]]]]
+                'password', ['validators' => [['Identical', true, ['admin_pass', 'messages' => tr('Passwords does not match.')]]]]
```
Comment (nuxwin) [inline gui/library/Functions/Input.php]: "does" is wrong there. We are talking about the passwordS and not the password. This issue is at several places in your changes.

Comment (DragonZX) [reply gui/library/Functions/Input.php]: Oups, my mistake. So, deny this pullrequest I'll make it back

**Comment by DragonZX** [comment]
Fixed

**Comment by DragonZX** [comment]
BTW in such case in my opinion it mustn't broke transifex, it's only fix for doubles, which are translated in the most cases. But it must be your desidion

**Comment by nuxwin** [comment]
@DragonZX 

You have changed several msgid in the code and thus, if I push them now and sync transifex, there will be several untranslated strings.

**Comment by DragonZX** [comment]
np, thanks for explanation.

**Comment by nuxwin** [comment]
@DragonZX 

BTW: https://github.com/i-MSCP/imscp/pull/86/files#diff-5c76ce57ea7868a7c395749514216737R187 ;) and maybe at other places. Fix that and I'll merge. There are only few modifications after all so, I can merge.

**Comment by DragonZX** [comment]
Fixed. Thanks for your assistance ;) I really need to get some sleep)

**Pull request merged**